Magic/src/main/java/com/elmakers/mine/bukkit/action/BaseShopAction.java

```diff
@@ -186,7 +190,9 @@ public SpellResult checkContext(CastContext context) {
             }
         }
 
+        //ToDO: maybe add check for skillAPI here too, I think here is where they check to cast spells
         // Check path requirements
+
```

Comment:
Please avoid any unrelated style changes.
It makes the diff more difficult to read.